src/runtime_src/core/edge/drm/zocl/zocl_bo.c

```diff
@@ -945,3 +945,19 @@ void zocl_clear_mem(struct drm_zocl_dev *zdev)
 
 	mutex_unlock(&zdev->mm_lock);
 }
+
+struct drm_gem_object *zocl_gem_import(struct drm_device *dev,
+				       struct dma_buf *dma_buf)
+{
+	struct drm_gem_object *gem_obj;
+	struct drm_zocl_bo *zocl_bo;
+
+	gem_obj = drm_gem_prime_import_dev(dev, dma_buf, dev->dev);
```

Comment:
any particular reason we don't want to use drm_gem_prime_import ?
and I think we'd better check the returned gem_obj, in the case of PTR_ERR(obj), we don't want to keep calling to_zocl_bo and set the flags field.

Reply:
You are right. We should use drm_gem_prime_import(), since we didn't want to attach dma-buf to another device. 
We should check PTR_ERR.